app/submitter/convert_payload_0_0_1.py

```diff
@@ -50,7 +53,7 @@ def convert_answers_to_payload_0_0_1(
             for answer_in_block in answers_in_block:
                 answer_schema = None
 
-                block = schema.get_block_for_answer_id(answer_in_block.answer_id)
+                block: ImmutableDict = schema.get_block_for_answer_id(answer_in_block.answer_id)  # type: ignore
```

Comment:
Some arguments for this `immutable dict`?

Reply:
ommitted due to being [str, Any]